src/automata/ChannelRequester.cpp

```diff
@@ -40,9 +40,16 @@ void ChannelRequester::onNextImpl(Payload request) noexcept {
         Base::generateRequest(remainingN);
       }
     } break;
-    case State::REQUESTED:
-      Base::onNext(std::move(request));
+    case State::REQUESTED: {
+      debugCheckOnNextOnCompleteOnError();
+      Frame_REQUEST_CHANNEL frame(
+          ConsumerMixin<Frame_RESPONSE>::streamId_,
+          FrameFlags_EMPTY,
+          std::move(request));
+      ConsumerMixin<Frame_RESPONSE>::connection_->outputFrameOrEnqueue(
```

Comment:
you dont have to specify ConsumerMixin<Frame_RESPONSE>::connection_->outputFrameOrEnqueue
you can just call connection_->outputFrameOrEnqueue